### PR TITLE
Readme examples improvements

### PR DESCRIPTION
**Pull request by MissingNO57**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,32 @@ To install with just support for Keras or Pytorch:
...
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/pytorch_mnist_diffpriv.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/pytorch_xray.py
+   ```
+- Or they can be accessed from colearn/examples by clonning colearn repo
+
+Run any example depending on what machine learning library support you've installed
+   ```bash
+   # for cloearn[keras] or colearn[all]
+   python3 keras_mnist.py
+   # for cloearn[pytorch] or colearn[all]
```
Comment (evsmithx) [inline README.md:59]: cloearn! It appears four times

File: docs/index.md [hunk 2]
```diff
@@ -44,10 +44,27 @@ To install with just support for Keras or Pytorch:
...
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/keras_mnist_diffpriv.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/keras_xray.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/mli_fraud.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/mli_random_forest_iris.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/pytorch_cifar.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/pytorch_covid.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/pytorch_mnist.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/pytorch_mnist_diffpriv.py
+   wget https://raw.githubusercontent.com/fetchai/colearn/master/examples/pytorch_xray.py
+   # Or they can be accessed from colearn/examples by clonning colearn repo
```
Comment (evsmithx) [inline docs/index.md:61]: cloning

Comment (evsmithx) [reply docs/index.md:61]: Also in the other file

**Review by evsmithx — CHANGES_REQUESTED** [review]
Only minor things!

File: README.md (hunk 1 above)
Comment (MissingNO57) [reply README.md:59]: clownlearn

**Review by JuanBesa — APPROVED** [review]

**Review by evsmithx — APPROVED** [review]

**Pull request merged**